docs/reference/tab-widgets/troubleshooting/data/restore-from-snapshot.asciidoc

```diff
@@ -0,0 +1,211 @@
++
+[role="screenshot"]
+image::images/troubleshooting/data/kibana-restore-snapshot-confirm.png[{kib} confirm snapshot restore,width=30%]
++
+. When the process will be complete you will see in the **Restore Status** tab the result of your restore.
++
+[role="screenshot"]
+image::images/troubleshooting/data/kibana-restore-snapshot-result.png[{kib} select resources to restore,align="center"]
++
+For more guidance on creating and restoring snapshots see
+<<snapshot-restore, this guide>>.
+
+//end::kibana-api-ex[]
+// end::cloud[]
+
+// tag::self-managed[]
+In order to restore the indices that are missing shards:
+
+. View the affected indices shards using the <<cat-indices,cat indices API>>.
++
+[source,console]
+----
+GET _cat/indices?v&health=red&h=index,status,health
+----
++
+The response will look like this:
++
+[source,console-result]
+----
+index                                status health
+.ds-my-data-stream-2022.06.17-000001 open   red
+kibana_sample_data_flights           open   red
+----
+// TEST[skip:illustration purposes only]
++
+The `red` health of the indices above indicates that these indices are missing primary shards,
+meaning they are missing data.
++
+. In order to restore the data we need to find a snapshot that contains these two indices.To find
+such a snapshot use the
+<<get-snapshot-api,get snapshot API>>.
++
+[source,console]
+----
+GET _snapshot/snap-repo/*?verbose=false
+----
+// TEST[skip:illustration purposes only]
++
+The response will look like this:
++
+[source,console-result]
+----
+{
+  "snapshots" : [
+    {
+      "snapshot" : "snapshot-20200617",                                     <1>
+      "uuid" : "dZyPs1HyTwS-cnKdH08EPg",
+      "repository" : "snap-repo",                                           <2>
+      "indices" : [                                                         <3>
+        ".apm-agent-configuration",
+        ".apm-custom-link",
+        ".ds-.logs-deprecation.elasticsearch-default-2022.06.17-000001",
+        ".ds-ilm-history-5-2022.06.17-000001",
+        ".ds-my-data-stream-2022.06.17-000001",
+        ".geoip_databases",
+        ".kibana-event-log-8.2.2-000001",
+        ".kibana_8.2.2_001",
+        ".kibana_task_manager_8.2.2_001",
+        "kibana_sample_data_ecommerce",
+        "kibana_sample_data_flights",
+        "kibana_sample_data_logs"
+      ],
+      "data_streams" : [ ],
+      "state" : "SUCCESS"                                                     <4>
+    }
+  ],
+  "total" : 1,
+  "remaining" : 0
+}
+----
+// TEST[skip:illustration purposes only]
++
+<1> The name of the snapshot.
++
+<2> The repository of the snapshot.
++
+<3> The indices backed up in the snapshot.
++
+<4> If the snapshot was successful.
+
+. The snapshot `snapshot-20200617` contains the two indices we want to restore.
+You might have multiple snapshots from which you could restore the target indices.The explanation in our case indicates the index allocation configurations are not correct.
```

Comment:
What does "The explanation in our case indicates the index allocation configurations are not correct"  mean?
Would this refer to the explain API? What index configurations would this refer to? 

Should we remove this or maybe change it to be more precise?

Reply:
This sentence was not intended, some weird mistake on my end, I removed it.